src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = element.querySelector('.js-Slideshow-slides'),
+    slides = element.querySelectorAll('.js-Slideshow-slide'),
+    nextTrigger = element.querySelector('.js-Slideshow-next'),
+    prevTrigger = element.querySelector('.js-Slideshow-prev'),
+    currentCountElement = element.querySelector('.js-Slideshow-current'),
+    totalCountElement = element.querySelector('.js-Slideshow-total'),
```

Comment:
@mrgerardorodriguez 

> Related: I was wondering...when using jQuery, a pattern arised to use the $ as a variable name prefix to denote if the element was a jQuery wrapped object.
>
> Might we want to follow a similar pattern here? Perhaps adding u as a prefix?

I don't have strong feelings about this. Honestly, I was never a fan of the `$` prefix, but adhered to it because it's such a widely accepted convention. As long as we're consistent. If we've already done this in a handful of other files, then I say do it. If this is the first file that uses the convention, then we should update all of the others using Umbrella to match it.